t04_refactored.py

```diff
@@ -218,9 +220,46 @@ def team_14_adv(username, delay):
 
 
 def team_15_adv(username, delay):
-    pass
+
+    """
+    # https://docs.google.com/document/d/1Xi3PVM3_64dIIpbPdPyi13yFwu7lCRqUi7PjU0Fy0hI/edit?usp=sharing
+    #Tobore Takpor
+    #Jack Ngog
+    """
+
+
     # TODO Add your code here
     return True
+
+from time import sleep
```

Comment:
Imports should be at the start of the program, and it should already be there. Remove this line.